conans/paths/__init__.py

```diff
@@ -57,5 +57,6 @@ def get_conan_user_home():
 SRC_FOLDER = "source"
 BUILD_FOLDER = "build"
 PACKAGES_FOLDER = "package"
+PACKAGE_INSTALL_FOLDER = "package_install"
```

Comment:
I don't love the name either (because of lengthy), but the truth is that "install" might have other meaning for users, and maybe users would be expecting something different from a ``install()`` method. ``package_install()`` makes it a bit more evident: it is not an install of dependencies (as the ``conan install`` command), but the installation of the current package. Then the folder is named after the method.

Reply:
My main concern here is packages that now will exceed the path length limit when previously they weren't. Maybe it is not `install`, but `deploy`, `frozen`, `installed`,... I still think we can make this move without breaking and I don't want this to fail just because the path is too long 😆

Reply:
Yeah, I'm on the same page, we need a name